src/editors/containers/EditorContainer/__snapshots__/index.test.jsx.snap

```diff
@@ -2,7 +2,9 @@
 
 exports[`EditorContainer component render snapshot: initialized. enable save and pass to header 1`] = `
 <div>
-  <ModalDialog.Header>
+  <ModalDialog.Header
+    className="border-bottom zindex-10"
```

Comment:
Can you make it a drop shadow instead of border line to match the expected design in paragon?